ui/StatusQ/src/StatusQ/Controls/StatusNavBarTabButton.qml

```diff
@@ -9,14 +9,6 @@ StatusIconTabButton {
     property alias tooltip: statusTooltip
     property Component popupMenu
 
-    signal clicked(var mouse)
```

Comment:
Just a note: this is evil! We already are an `AbstractButton` which has its own `clicked()` signal with a different signature; this one is shadowing it